.changelog/4006.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Fixed an issue where validate would state that it failed, even though it shouldn't, and then it would display an empty list of content items.
```

Comment:
```suggestion
- description: Fixed an issue where validate would state that it failed, even though it shouldn't, and then it would display an empty list of content items when running on specific validations.
```